apis/itemsetapi/itemsetapi.yaml

```diff
@@ -0,0 +1,840 @@
+                    "lastPublishedBy": "89cf1a7e-dfd3-46c9-a428-d37e9a2bc001"
+                  }
+                }
+              }
+          expectedResponse:
+            x-allowExtraHeaders: true
+            x-bodyMatchMode: NONE
+            x-arrayOrderedMatching: false
+            x-arrayCheckCount: false
+            x-matchResponseSchema: true
+            headers: {}
+          x-testShouldPass: true
+          x-testEnabled: true
+          x-testName: Publish ItemSet
+          x-testDescription: ''
+      x-operation-settings:
+        CollectParameters: false
+        AllowDynamicQueryParameters: false
+        AllowDynamicFormParameters: false
+        IsMultiItemSetStreaming: false
+  /retire/{ItemSet_ID}:
+    delete:
+      description: >-
+        This API is associated with retiring the itemset on the Sunbird Platform.
+          - The endpoint for **Retire ItemSet** is `/retire/{ItemSet_ID}`
+          - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.
+
+      summary: Retire ItemSet
+      tags:
+        - ItemSet APIs
+      operationId: Retire ItemSet
+      produces:
+        - application/json
+      parameters:
+        - in: path
+          name: ItemSet_ID
+          type: string
+          required: true
+          description: 'Please append a valid ItemSet Id To the Request URL'
+        - name: Body
+          in: body
+          required: true
+          description: >-
+           The body parameter refers to the format of the request, the request body (payload) takes the following media types to make the payload:'
+            
+            - JSON 
+            - XML data
+          
+            The body is the representation of the resource object for deleting item set
+          schema:
+            $ref: '#/definitions/RetireItemSetRequest'
+        - name: Content-Type
+          in: header
+          required: true
+          type: string
+          description:  >-
+            The ItemSet Type entity is the media type of the resource.Possible media types can be:- 
+              - Application/json
+        - name: X-Authenticated-Userid
+          in: header
+          required: true
+          type: string
+          description: 'Represents the User ID registered to execute the API.'
+        - name: Authorization
+          in: header
+          required: true
+          type: string
+          description: 'To make use of the API, you require authorization. Raise a request to the administrator for the use of the API. You will receive the authorization key. Specify the key received, here.'
+      responses:
+        '200':
+          description: >-
+            SUCCESS. The **Delete ItemSet** operation was successful!
+          schema:
+            $ref: '#/definitions/ItemSetCreateResponse'
+        '400':
+          description: >-
+            BAD REQUEST. The **Delete ItemSet** operation failed. You may have missed input for a mandatory parameter.
```

Comment:
The name of the operation ** Delete ItemSet** is not consistent with the name of the API end point. It should be consistent.